packages/nx-go/src/generators/library/generator.ts

```diff
@@ -54,7 +54,7 @@ export default async function libraryGenerator(
   if (isGoWorkspace(tree)) {
     createGoMod(
       tree,
-      `${options.npmScope}/${options.moduleName}`,
+      `${options.npmScope}/${options.name}`,
```

Comment:
**suggestion:** This should be the project root.

This way the import path would have the same path as the file path to the project folder. Not sure if this how others build out their go workspaces but this is how we define our module names in `go.mod` files. 

Its not a big deal if we want to leave it but if we execute a command like the following:

```bash
nx g @nx-go/nx-go:library workflows --directory libs/feature/admin
```

It will create a go.mod file like so:

```txt
module [npmScope]/workflows

go 1.22
```

If I want to add a workflows module to a different feature this will create a collision.

Reply:
I know in the original issue the example used the name as a path but I wanted to check what happens when a directory is provided.

Reply:
Awesome suggestion, indeed my solution was not ideal. I will change that, thank you for the feedback!